exercises/luhn/canonical-data.json

```diff
@@ -29,6 +34,36 @@
       "description": "valid strings with a non-digit added become invalid",
       "input": "046a 454 286",
       "expected": false
+    },
+    {
+      "description": "punctuation is not allowed",
+      "input": "055-444-285",
+      "expected": false
+    },
+    {
+      "description": "symbols are not allowed",
+      "input": "055£ 444$ 285",
```

Comment:
can I get a confirmation on whether the £ is in ASCII? I am seeing bytes 0xc2 0xa3 for it

Reply:
It is not.